README.md

```diff
@@ -25,13 +25,40 @@ Below you will find the steps to deploy an application as a **StarlingX App**.
     - [Other files](#other-files)
 
 ## Config files precedence
-When packaging an application as an starlingx application, you can place specific application's
-in multiple places (even though you probably shouldn't). The files where you can place these configurations
-will be listed below by their order of precedence in descending sorting.
-1. user-overrides.yaml - passed as in input after an application has been uploaded through the command "system helm-override-update --values _user-overrides.yaml_" ...
-2. _application-name_/plugins/**/helm/*.py - configurations passed into the `get_overrides` function
-3. _application-name_/fluxcd-manifests/**/*-static-overrides.yaml - passed to the fluxcd static overrides file
-4. _helm-chart_/values.yaml
+
+When packaging an application as a StarlingX App, there are multiple places
+where you can place your app configuration.
+Below you'll find a list of those places in descending order by their
+precedence (every item on the list supersedes the one(s) below):
+
+1. User overrides file;
```

Comment:
Maybe just 'User overrides"? The file by itself does nothing, the user must apply the overrides manually

Reply:
```suggestion
1. User overrides;
```